tests/suites/0_stateless/12_time_travel/12_0004_time_travel_select_at.sh

```diff
@@ -22,5 +26,14 @@ echo "select count(*) from t12_0004 at (snapshot => '$SNAPSHOT_ID')" | $MYSQL_CL
 echo "planner_v2: counting the data set of first insertion, which should contains 2 rows"
 echo "set enable_planner_v2 = 1;select count(t.c) from t12_0004 at (snapshot => '$SNAPSHOT_ID') as t" | $MYSQL_CLIENT_CONNECT
 
+
+# Get a time point at/after the first insertion.
+# Cannot directly use the timestamp of first insertion, because the time precision of snapshot is higher than timestamp(6). So +1μs.
+
+TIMEPOINT=$(echo "select cast(timestamp as BIGINT) + 1 from fuse_snapshot('default', 't12_0004') where row_count=2" | $MYSQL_CLIENT_CONNECT)
```

Comment:
I found the precision of snapshot timestamp is nanoseconds, and I can only get `TIMESTAMP(6)` from `fuse_snapshot()`. So add 1 microsecond to the result, is it OK?

Reply:
It seems to be a bug or at least incoherence of components, would you please open an issue for it later?